src/MessageHandler.py

```diff
@@ -0,0 +1,112 @@
+import json
+import urllib2
+from src.MessageRouter import MessageRouter
+
+
+def new_message_handler(config):
+    def _new_command_handlers():
+        return {
+            'casts': _new_casts_handler(),
+            'streams': new_get_streams_response(fetch_url_contents),
+            'nyan': lambda message: ['~=[,,_,,]:3']
```

Comment:
Not sure I understand. Is what you suggest marking the message handling functions in some way so they are automatically detected as message handler? If so, I'm back to my concerns about configuration (which message handlers are used) needing to be dynamic, not static.